### ISSUE TITLE
Impossible new checks from v0.2.5

### ISSUE THREAD

**Issue by G4M3RL1F3** [LittleCube-hax/MMRecompRando#38]
Some of the new locations that were added in mod release v0.2.5 do not send. Here's the list:
- `West Clock Town Bank 500 Rupees`: It still gives a blue rupee.
- `Termina Bio Baba Grotto HP`: There's nothing at the bottom of the water. It's been like that for a while too, even before v0.2.5, but didn't matter up until now.
- `Swamp Shooting Gallery 2180 Points`: I played the game a second time to see if I had a "Clock Town Shooting Gallery" moment, but it still didn't send. It just gave me a magic bean instead.
- `Deku Palace Bean Seller`: Before v0.2.5, he was giving a free magic bean. Now, he doesn't give an item and only sells beans instead.

**Comment by ThatHypedPerson** [comment]
I just checked all of these locations real quick and they seem to work? My assumption is that you may have forgotten to update something (such as the apworld), but I may be wrong.

Also I'm not sure if you knew, but to get `Termina Bio Baba Grotto HP` you need to knock down a beehive within that grotto first before the Heart Piece/check appears.

**Comment by G4M3RL1F3** [comment]
I updated both the apworld and the mods. For the Bio Baba grotto, I knocked down the beehives as well, but nothing came out of them.

**Comment by LittleCube-hax** [comment]
@G4M3RL1F3 Can you send your spoiler for the seed?

**Comment by G4M3RL1F3** [comment]
[AP_22583564835504898859_Spoiler.txt](https://github.com/user-attachments/files/17685704/AP_22583564835504898859_Spoiler.txt)

**Comment by LittleCube-hax** [comment]
Okay, it didn't shuffle `Nothing` there. I'll plando Blast Mask onto it and take a look.

**Comment by G4M3RL1F3** [comment]
Okay, I figured it out. The issue was on my end specifically.

Whenever there's a new mod release, I export on the mods folder, letting it replace everything inside. There are those two files called `mm_recomp_rando-0.2.x`, one with the extension `.dll` and the other `.nrm`. When 0.2.5 released, those two files' version changed from `0.2.3` to `0.2.5`. So with my method of updating, both versions were still in the mods folder, creating some issues.

I deleted everything inside the mods folder, installed the 0.2.5 mods and tested a run. The checks that were previously impossible for me were now possible!
